user-defined-variables.md

```diff
@@ -179,4 +179,8 @@ SELECT @col FROM t;
 +------+
 ```
 
+## MySQL 兼容性
+
+除`SELECT ... INTO <variable>`外，MySQL和TiDB支持的语法相同。
```

Comment:
Please always leave a space between Chinese and English (except for Chinese punctuations).
```suggestion
除 `SELECT ... INTO <variable>` 外，MySQL 和 TiDB 支持的语法相同。
```